pkgs/top-level/all-packages.nix

```diff
@@ -7227,6 +7227,8 @@ in
   adoptopenjdk-bin = adoptopenjdk-hotspot-bin-11;
   adoptopenjdk-jre-bin = adoptopenjdk-jre-hotspot-bin-11;
 
+  adoptopenjdk-icedtea-web = icedtea_web;
```

Comment:
We should `callPackage` the file as `adoptopenjdk-icedtea-web` and add an alias to `aliases.nix`
with
```
icedtea_web = adoptopenjdk-icedtea-web; # added YYYY-MM-DD

Reply:
I see. However at the moment the package is defined in `development/compilers/icedtea-web`, should I rename the folder too?
In this case, I'd rather submit a new PR with the package name change.

So, should I go ahead and rename everything in this PR, or would it be better merge this PR with the CE fixes and have a new PR with the package rename?
My preference goes to the latter option, what do you think?